.github/workflows/build.yml

```diff
@@ -0,0 +1,106 @@
+
+name: build
+
+on:
+  # Triggers the workflow on push or pull request events but only for the master branch
+  #push:
+  #  branches: [ master ]
+  #pull_request:
+  #  branches: [ master ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        # for single tests (comment all the other defs)
+        #os: ["opensuse/leap:15.3"]
+        #pkgs: ["clang gcc10 gcc10-c++"]
+        #vars1: ["CC=clang CXX=clang++"]
+        os: ["opensuse/leap:15.3"]
+        pkgs: ["gcc10 gcc10-c++"]
+        vars1: ["CC=gcc-10 CXX=g++-10"]
+        include:
```

Comment:
Could you also add Alpine in order to test this on a musl-based distro? See https://gitlab.alpinelinux.org/alpine/aports/-/blob/master/community/android-tools/APKBUILD#L11 regarding dependencies.